src/components/document/sort-work-document-area.tsx

```diff
@@ -55,6 +55,12 @@ export const SortWorkDocumentArea: React.FC<IProps> = observer(function SortWork
     );
   };
 
+  const handleCloseButtonClick = () => {
+    if (persistentUI.openSubTab) {
+      persistentUI.closeSubTabDocument(persistentUI.activeNavTab, persistentUI.openSubTab);
+    }
+  };
+
```

Comment:
Because this is supposed to apply to all documents on the left side. This needs to be duplicated in DocumentArea. Ideally we'd refactor SortWorkDocumentArea and DocumentArea to extract the comment parts, but since we are supposed to wrap this work up, it seems OK to just copy and past the close button code to both places.